mops-torch/src/opsa.cpp

```diff
@@ -41,10 +41,28 @@ torch::Tensor OuterProductScatterAdd::forward(
                     torch_to_mops_1d<int32_t>(indices_output));
             });
     } else {
+
+#ifndef MOPS_CUDA_ENABLED
         C10_THROW_ERROR(
             ValueError,
             "outer_product_scatter_add is not implemented for device " +
```

Comment:
I feel like we need a different error message here now. Something like "mops wasn't compiled with CUDA support"?